test/Access_Control/Curio/README.md

```diff
@@ -0,0 +1,198 @@
+# Curio Privilege Escalation
+- **Type:** Exploit
+- **Network:** Ethereum 
+- **Total lost**: ~16MM USD 
+- **Category:** Access Control
+- **Vulnerable contracts:**
+- - [Curio Chief](https://etherscan.io/address/0x579A3244f38112b8AAbefcE0227555C9b6e7aaF0#code)
+- - [Curio Pause](https://etherscan.io/address/0x1e692eF9cF786Ed4534d5Ca11EdBa7709602c69f#code)
+
+- **Attack transactions:**
+- - [Privilege escalation and mint](https://etherscan.io/tx/0x4ff4028b03c3df468197358b99f5160e5709e7fce3884cc8ce818856d058e106)
+  
+- **Attacker Addresses**: 
+- - Attacker EOA: [0xdaAa6294C47b5743BDafe0613d1926eE27ae8cf5](https://etherscan.io/address/0xdaAa6294C47b5743BDafe0613d1926eE27ae8cf5)
+- - Attacker Contract (verified): [0x1E791527AEA32cDDBD7CeB7F04612DB536816545](https://etherscan.io/address/0x1E791527AEA32cDDBD7CeB7F04612DB536816545)
+  
+- **Attack Block:**:  `19,498,911`  
+- **Date:** Mar 23, 2024
+- **Reproduce:** `forge test --match-contract=Exploit_Curio -vvv`
+
+## Step-by-step 
```

Comment:
The step-by-step is weird because it describes what happens in the contract, instead of the step-of-step for the attack, as usual.